utils/githooks/pre-commit

```diff
@@ -1,21 +1,47 @@
 #!/bin/bash
 
-# A git hook to validate  and correct the copyright date in source files.
+# A git hook to validate and correct the copyright date in source files.
 if [ -n "${NO_PRE_COMMIT:=""}" ]; then
-    exit 0
+	exit 0
 fi
 
 regex='(^[[:blank:]]*[\*/]*.*)((Copyright[[:blank:]]*)([0-9]{4})(-([0-9]{4}))?)([[:blank:]]*(Intel.*$))'
 year=$(date +%Y)
 errors=0
+targets=(
+	# code source files
+	'*.c'
+	'*.h'
+	'*.go'
+	'*.py'
+	'*.proto'
+	'*.java'
+	'*.yml'
+	'*.yaml'
+	'*.sh'
+	'*.bash'
+	# DevOps files
+	'Makefile'
+	'Jenkinsfile'
+	'SConscript'
```

Comment:
Could also add SConstruct here.   It's the main SCons file in the main directory

Reply:
Please wait until you have two reviews before adding gatekeeper for review

Reply:
Thanks for the information on gatekeeper management.
I will add the SConstruct file

Reply:
Do you think, I should add more reviewers: is there a best practice on the number of reviewers ?